topdown/numbers_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+package topdown
+
+import (
+	"testing"
+)
+
+func TestBuiltinNumbersRange(t *testing.T) {
+	cases := []struct {
+		note string
+		stmt string
+		exp  string
+	}{
+		{
+			note: "one",
+			stmt: "p = x { x := numbers.range(0, 0) }",
+			exp:  "[0]",
+		},
+		{
+			note: "ascending",
+			stmt: "p = x { x := numbers.range(-2, 3) }",
+			exp:  "[-2, -1, 0, 1, 2, 3]",
+		},
+		{
+			note: "descending",
+			stmt: "p = x { x := numbers.range(2, -3) }",
+			exp:  "[2, 1, 0, -1, -2, -3]",
+		},
+		{
+			note: "precision",
+			stmt: "p { numbers.range(49649733057, 49649733060, [49649733057, 49649733058, 49649733059, 49649733060]) }",
+			exp:  "true",
+		},
```

Comment:
We've been adding in negative tests, in particular for the arguments, should we add a few here too? If anything just catching like if a float number is passed in.

Reply:
Added tests for floating point error condition.